events/meeting_handler.py

```
@@ -0,0 +1,57 @@
+            if ws is self.websocket:
+                continue
+            await ws.send(json.dumps(message))
+
+    def _join(self, code):
+        if code not in self.rooms:
+            self.rooms[code] = [] # Set instead of list ?
+        
+        self.rooms[code].append(self.websocket) # Verify that it's not present before adding ?
+        self.joined = code
+
+    def _leave(self):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_leave`

Add return type annotation: `None`

(ANN202)